spring-cloud-task-stream/src/main/java/org/springframework/cloud/task/launcher/TaskLauncherSink.java

```diff
@@ -60,7 +65,10 @@ public void taskLauncherSink(TaskLaunchRequest request) {
 	private void launchTask(TaskLaunchRequest taskLaunchRequest) {
 		Assert.notNull(taskLauncher, "TaskLauncher has not been initialized");
 		logger.info("Launching Task for the following resource " + taskLaunchRequest);
-		Resource resource = delegatingResourceLoader.getResource(taskLaunchRequest.getUri());
+		MavenProperties mavenProperties = new MavenProperties();
+		Map<String, MavenProperties.RemoteRepository> remoteRepositoryMap = new HashMap<>();
```

Comment:
did you intend to add a repo entry here (something would need to be configurable as a property of the launcher app itself)?... otherwise this cannot retrieve an app from anywhere except "local" and this app could very likely be running in an environment that has nothing published to its local repo